tests/functional/ui/test_health_overview.py

```
@@ -50,6 +54,66 @@
 
+def get_alert_params():
+    """
+    Get alert parameters based on OCS version
+
+    Returns:
+        list: List of tuples (alert_name, alert_yaml)
+    """
+    try:
+        ocs_version = get_ocs_version_from_csv(only_major_minor=True)
+        if ocs_version is None:
+            raise ValueError("get_ocs_version_from_csv returned None")
```

Ruff (0.15.14)

[warning] Abstract `raise` to an inner function

(TRY301)